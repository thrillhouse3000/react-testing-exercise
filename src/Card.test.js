import { render, fireEvent } from "@testing-library/react";
import Card from "./Card";

test('it renders', () => {
    render(<Card/>)
})

test('snapshot matches', () => {
    const {asFragment} = render(<Card caption='test' src='image1' currNum={1} total={3} />)
    expect(asFragment()).toMatchSnapshot()
})