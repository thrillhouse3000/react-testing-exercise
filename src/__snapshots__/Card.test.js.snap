// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot matches 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      test
    </h4>
    <img
      alt="test"
      class="Card-image"
      src="image1"
    />
    <small
      class="Card-small"
    >
      Image 1 of .
    </small>
  </div>
</DocumentFragment>
`;
